src/geocoders/here.js

```diff
@@ -28,6 +28,36 @@ export default {
       this.getJSON(this.options.geocodeUrl, params, cb, context);
     },
 
+    geocodeSuggestion: function(query, cb, context) {
+      var params = {
+        locationid: query,
+        gen: 9,
+        app_id: this.options.app_id,
+        app_code: this.options.app_code,
+        jsonattributes: 1
+      };
+      params = L.Util.extend(params, this.options.geocodingQueryParams);
+      this.getJSON(this.options.geocodeUrl, params, cb, context);
+    },
+
+    suggest: function(query, cb, context) {
+      var _this = this;
+      var params = {
+        query: query,
+        app_id: this.options.app_id,
+        app_code: this.options.app_code,
+        country: this.options.country
```

Comment:
The (optional) country should be set via `options.suggestQueryParams` (cf. `geocodingQueryParams`, `reverseQueryParams`)